trie/trie.go

```diff
@@ -593,6 +593,19 @@ func (t *Trie) resolveAndTrack(n hashNode, prefix []byte) (node, error) {
 	return mustDecodeNode(n, blob), nil
 }
 
+// resolveWithoutTrack loads node from the underlying store with the given node hash
+// and path prefix.
+func (t *Trie) resloveWithoutTrack(n node, prefix []byte) (node, error) {
```

Comment:
there's a spelling error here, but aside from that, why do you need a new method insetad of using `resolveAndTrack` ?

Reply:
Because recording these nodes in the tracer doesn't make any sense.